plugin/pkg/scheduler/extender.go

```diff
@@ -92,12 +92,13 @@ func NewHTTPExtender(config *schedulerapi.ExtenderConfig, apiVersion string) (al
 }
 
 // Filter based on extender implemented predicate functions. The filtered list is
-// expected to be a subset of the supplied list.
-func (h *HTTPExtender) Filter(pod *api.Pod, nodes *api.NodeList) (*api.NodeList, error) {
+// expected to be a subset of the supplied list. The failedNodesMap is expected to
+// list the filtered out nodes information.
+func (h *HTTPExtender) Filter(pod *api.Pod, nodes *api.NodeList) (*api.NodeList, schedulerapi.FailedNodesMap, error) {
 	var result schedulerapi.ExtenderFilterResult
 
 	if h.filterVerb == "" {
-		return nodes, nil
+		return nodes, schedulerapi.FailedNodesMap{}, nil
```

Comment:
The failed nodes is relative to the first returned value(filteredNodes), when there are no filterVerb, the filteredNodes is all the nodes, and the failed nodes is a empty map, not nil, I think.
And there are the same way in findNodesThatFit function. I think it's the same consideration.